azext_edge/tests/edge/support/create_bundle_int/test_mq_int.py

```diff
@@ -78,6 +80,7 @@ def test_create_bundle_mq(init_setup, tracked_files, mq_traces):
     check_workload_resource_files(
         file_objs=file_map,
         expected_workload_types=expected_workload_types,
-        prefixes=["aio-broker", "aio-dmqtt"],
+        prefixes=["aio-broker", "aio-dmqtt", "otel-collector-service"],
```

Comment:
Is aio-dmqtt still necessary ?

Reply:
I haven't seen it but I want to keep it in just in case it pops up again...